gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -169,6 +169,7 @@
 
         lifecycleManager.update(finalPayload, function (response) {
             if (response.success) {
+                delete finalPayload.lifecycleInstructions;
```

Comment:
I dont believe this will affect simon, but I guess this change was from before our talk at last arch. meeting... 

The only reason I delete these is to avoid the 'lifecycleInstructions' block being attached to the sessionState in the next line. When doing this patch I got confused that there was so much crap attached to sessionState - and lifecycleInstructions is never used later on, so there was no reason for having it there.. It is only used in the "finalPayload" as instructions on how to configure the system.. once configured, the applied settings are stored in 'activeConfiguration'